### PR TITLE
Pp 466 fix missed delegate call enter manually

### PR DESCRIPTION
**Pull request by zladzeyka**
- Add `GiniCaptureResultsDelegate.giniCaptureDidEnterManually` missing calls on the No results and Error screens
 - Add `GiniCaptureResultsDelegate.giniCaptureDidCancelAnalysis` missing calls on the cancel tap on the Camera, Analysis and No results and Error screens
 - Example app: add alert on the `giniCaptureDidEnterManually` and dismiss call on `giniCaptureDidCancelAnalysis`

### CODE REVIEW

**Review by a-szotyori — APPROVED** [review]
Nice! 👍

File: BankSDK/GiniBankSDKExample/GiniBankSDKExample/Screen API/ScreenAPICoordinator.swift [hunk 1]
```diff
@@ -149,13 +149,26 @@ final class ScreenAPICoordinator: NSObject, Coordinator, UINavigationControllerD
         configuration.cleanup()
         delegate?.screenAPI(coordinator: self, didFinish: ())
     }
+
+    private func showAlertOnDidEnterManually(){
+        let alert = UIAlertController(title: "GiniCaptureResultsDelegate was called",
+                                      message: "",
```
Comment (ValentinaIancu-Gini) [inline BankSDK/GiniBankSDKExample/GiniBankSDKExample/Screen API/ScreenAPICoordinator.swift:155]: [suggestion]: we can ` message` to nil since is optional.

**Comment by ValentinaIancu-Gini** [comment]
@zladzeyka  Thank you for helping with this. I left a tiny suggestion.

**Review by ValentinaIancu-Gini — APPROVED** [review]
@zladzeyka Thank you!

**Pull request merged**